crypto/objects/o_names.c

```diff
@@ -157,11 +174,13 @@ const char *OBJ_NAME_get(const char *name, int type)
 {
     OBJ_NAME on, *ret;
     int num = 0, alias;
+    const char *value = NULL;
 
     if (name == NULL)
         return (NULL);
-    if ((names_lh == NULL) && !OBJ_NAME_init())
+    if (!OBJ_NAME_init())
         return (NULL);
+    CRYPTO_THREAD_read_lock(lock);
```

Comment:
Updated - it's kind of awful. Does anyone actually use `OPENSSL_LH_stats_bio`? It'd be great to remove the statistics tracking entirely and have that thing print a message about stats no longer being recorded.